### DISCUSSION TITLE
How to interface between an Android App and a Pico?

### DISCUSSION THREAD

**Discussion by JoePShoulak** [fhdm-dev/scoppy#64] [Q&A]
I love this project, it's inspired me to try to develop something on my own using an android app. I had trouble dissecting the code, can someone point me in the right direction on interfacing between those devices?

**Comment by topherbuckley** [comment]
I'm also interested in this, and I'm guessing the search starts [here](https://github.com/fhdm-dev/scoppy-pico/tree/master/pico/scoppy-usb)
